README.md

```diff
@@ -32,4 +32,4 @@ monitoring_nexporter_image_version: latest
 ## Security
 
 This role exposes through an HTTP server lots of data that can be potentially exploited. By default, it listens to a loopback adress, not public from the internet. However, you probably want an external Prometheus server to fetch this data periodically. To this end, and to protect the data, some sort of authentication from the Prometheus server against the host is needed.
-One approach is to leave the exporters binding to localhost and then set up a reverse proxy before them with Basic Authentication, using Nginx. The management of this set up and of the keys implied are out of the scope of this role.
\ No newline at end of file
+One approach is to leave the exporters binding to localhost and then set up a reverse proxy before them with Basic Authentication, using Nginx. The management of this set up and of the keys implied are out of the scope of this role.
```

Comment:
Not sure about where this comes from :thinking:

Reply:
What do you mean? I added this section to say "hey, bear in mind that this is not gonna work out of the box" and "one way to complement this role is this".

Do you prefer another example? Not to explain any example?

Reply:
I mean... I didn't touch these lines... and I see no DIFF...